tools/blender/2.9x/pie_addon/pie_import.py

```diff
@@ -478,9 +530,16 @@ def getTypeFlags(num):
             else:
                 objProp.tcMask = False
 
-    def pie_import(self, scene, pieDir, pieMesh):
+    def pie_import_quick(self, scene, pieDir, pieMesh):
         self.scene = scene
 
         pieParse = self.pie_parse(pieDir + '\\\\' + pieMesh)
 
         self.pie_generateBlenderObjects(pieParse)
+
+    def pie_import(self, scene, pieDir, pieMesh):

```

Comment:
@Solstice245:
Should this be:

```suggestion
    def pie_import(self, scene, pieDir):
```

?

LGTM is complaining about the call:
`importer.pie_import(context.scene, self.properties.filepath)`
in `__init__.py` only passing 2 parameters.